src/main/java/net/neoforged/neoforge/transfer/handlers/resources/IResourceHandler.java

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.transfer.handlers.resources;
```

Comment:
I suggest we flatten most of this PR into the `net.neoforged.neoforge.transfer.handlers` package.

Reply:
> I suggest we flatten most of this PR into the `net.neoforged.neoforge.transfer.handlers` package.

If at all flattening should happen after the last slice, it is very hard to determine that right now.

Please leave at is, flattening will take 30 seconds most, at the end.